content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Istio DNS Certificate Management
+description: Shows how to provision and manage DNS certificates in Istio.
+weight: 90
+keywords: [security,certificate]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+By default, the DNS certificates used by the webhooks of Galley and the sidecar
+injector are provisioned and managed by Citadel, which is a large component
+that maintains its own signing key and also acts as a CA for Istio.
+This task shows how to provision and manage DNS certificates for Istio control
+plane components through Chiron: a lightweight component that signs certificates
+using Kubernetes CA APIs without maintaining its own private key.
+Using this feature has the following advantages:
+
+* No dependency on Citadel for DNS certificates and is more lightweight than Citadel.
+
+* Unlike Citadel, this feature doesn't require maintaining a private signing key, which enhances security.
+
+* Simplified root certificate distribution to TLS clients. Clients no longer need to wait for Citadel to generate and distribute its CA certificate.
+
+## Before you begin
+
+* Install Istio through `istioctl` with DNS certificates configured. This feature requires Istio 1.4+ and
```

Comment:
This doc set is for 1.4, so no need to mention the version requirement.

Reply:
Done.